sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+|---------------|-------------|
+| [`ALTER PLACEMENT POLICY`](/sql-statements/sql-statement-alter-placement-policy.md) | Modifies a placement policy. |
+| [`ALTER RANGE`](/sql-statements/sql-statement-alter-range.md) | Modifies the range of a placement policy. |
+| [`CREATE PLACEMENT POLICY`](/sql-statements/sql-statement-create-placement-policy.md) | Creates a new placement policy. |
+| [`DROP PLACEMENT POLICY`](/sql-statements/sql-statement-drop-placement-policy.md) | Drops an existing placement policy. |
+| [`SHOW CREATE PLACEMENT POLICY`](/sql-statements/sql-statement-show-create-placement-policy.md) | Shows the CREATE statement for a placement policy. |
+| [`SHOW PLACEMENT FOR`](/sql-statements/sql-statement-show-placement-for.md) | Shows placement rules for a specific table. |
+| [`SHOW PLACEMENT LABELS`](/sql-statements/sql-statement-show-placement-labels.md) | Shows available placement labels. |
+| [`SHOW PLACEMENT`](/sql-statements/sql-statement-show-placement.md) | Shows placement rules. |
+
+## Resource groups
+
+<CustomContent platform="tidb">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md) | Modifies a resource group. |
+| [`CALIBRATE RESOURCE`](/sql-statements/sql-statement-calibrate-resource.md) | Calibrates system resources. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md) | Modifies a resource group. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+## Utility statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`DESC`](/sql-statements/sql-statement-desc.md) | Describes the structure of a table. |
```

Comment:
```suggestion
| [`DESC`](/sql-statements/sql-statement-desc.md) | An alias to `EXPLAIN`, which shows the execution plan of a query without executing it. |
```

Reply:
Both `DESC`/`DESCRIBE` and `EXPLAIN` do two different things:


Describe the structure of a table:
```
mysql> DESCRIBE t1;
+-------+---------+------+------+---------+-------+
| Field | Type    | Null | Key  | Default | Extra |
+-------+---------+------+------+---------+-------+
| id    | int(11) | NO   | PRI  | NULL    |       |
+-------+---------+------+------+---------+-------+
1 row in set (0.00 sec)
```

Show the explain plan of a query:
```
mysql> EXPLAIN SELECT 1;
+-------------------+---------+------+---------------+---------------+
| id                | estRows | task | access object | operator info |
+-------------------+---------+------+---------------+---------------+
| Projection_3      | 1.00    | root |               | 1->Column#1   |
| └─TableDual_4     | 1.00    | root |               | rows:1        |
+-------------------+---------+------+---------------+---------------+
2 rows in set (0.00 sec)
```

While these are aliases and can be all be used for both of these operations, it is more common to use `DESC`/`DESCRIBE` to list the structure of a table and to use `EXPLAIN` with execution plans.

Note that `SHOW COLUMNS FROM` can also be used to inspect the table structure, but not to get execution plans.

Reply:
Thanks for pointing that out. I copied the descriptions from [DESC | TiDB SQL Statement Reference](https://docs.pingcap.com/tidb/dev/sql-statement-desc) and [DESCRIBE | TiDB SQL Statement Reference](https://docs.pingcap.com/tidb/dev/sql-statement-describe)😂. Perhaps, another PR is needed to fix the descriptions in these docs.

Reply:
Let's list `DESC` as alias for `DESCRIBE` and document this as a statement to get a table description.

Then `EXPLAIN` can be documented as statement for getting an explain plan.